solc_select/__main__.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+from .solc_select import *
+
+def solc_select():
+    INSTALL_VERSIONS = "INSTALL_VERSIONS"
+    USE_VERSION = "USE_VERSION"
+    SHOW_VERSIONS = "SHOW_VERSIONS"
+
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(help='Allows users to install and quickly switch between Solidity compiler versions')
+    parser_install = subparsers.add_parser('install', help='list and install available solc versions')
+    parser_install.add_argument(INSTALL_VERSIONS, help='specific versions you want to install "0.4.25" or "all"', nargs="*", default=list(), type=valid_install_arg)
+    parser_use = subparsers.add_parser('use', help='change the version of global solc compiler')
+    parser_use.add_argument(USE_VERSION, help='solc version you want to use (eg: 0.4.25)', type=valid_version)
+    parser_use = subparsers.add_parser('versions', help='prints out all installed solc versions')
+    parser_use.add_argument(SHOW_VERSIONS, nargs='*', help=argparse.SUPPRESS)
+
+    args = vars(parser.parse_args())
+
+    if args.get(INSTALL_VERSIONS) is not None:
+        versions = args.get(INSTALL_VERSIONS)
+        if versions == []:
+            print("Available versions to install:")
+            for version in reversed(sorted(get_installable_versions())):
```

Comment:
I think the versions should be listed in reverse order, with the newest versions at the bottom. The user is going to most likely be interested in a newer version, so this way they won't have to scroll up unless they want an ancient version.

```suggestion
            for version in sorted(get_installable_versions()):
```

Reply:
Fixed